File: Arrays/Question 12/opposite_count.js
function opposite_count(arr){
    let count = 0;
    for(let i =0; i<arr.length; i++){
        for(let j=0; j<arr.length; j++){
          if (j > i){
            if(arr[i] + arr[j] === 0){
              count++;
            }
          }
        }
    }
  
    return count;
  }
  
  console.log(opposite_count([ 2, 5, 11, -5, -2, 7 ]));
  console.log(opposite_count([ 21, -23, 24 -12, 23 ]));

File: Arrays/Question 12/opposite_count.rb
def opposite_count(nums)
    count = 0
  
    nums.each_with_index do |num1, idx1|
      nums.each_with_index do |num2, idx2|
        if num1 + num2 == 0 && idx2 > idx1
          count += 1
        end
      end
    end
  
    return count
  end
  
  puts opposite_count([ 2, 5, 11, -5, -2, 7 ]) # => 2
  puts opposite_count([ 21, -23, 24 -12, 23 ]) # => 1